macros/HTMLRef.ejs

```diff
@@ -108,6 +122,10 @@ var resultAPI = [];
             <li><%- template("domxref", [resultAPI[aTitle]]) %></li>
   <%    } 
     } %>
+    <% if (env.slug.includes("/HTML/Element/input")) { %>
+   <li data-default-state="<%=state('Input_types')%>"><a href="/<%=env.locale%>/docs/Web/HTML/Element/input#Form_<input>_types"><%-text['Input_types']%></a>
```

Comment:
This can just be "open" as the state function won't be used at all (you won't call {{HTMLRef("Input_types")}}). You just want the input types to be there and expanded when you are under the input tree, which is already dealt with by  `<% if (env.slug.includes("/HTML/Element/input")) { %>`

So, please change this:

```diff
-   <li data-default-state="<%=state('Input_types')%>">
+   <li data-default-state="open">
```

Reply:
Oh yeah. That's right. I gotcha.